client/src/screens/events/Response.js

```diff
@@ -172,6 +175,31 @@ class EventResponse extends Component {
     });
   }
 
+  handleDetailChange = (answer) => {
+    this.setState({
+      detail: answer !== '' ? answer : null,
+    });
+  }
+
+  handleDetail = () => {
+    this.setState({
+      detailModal: false,
+    }, () => this.submitEventResponse());
```

Comment:
in theory this could be:
```
}, this.submitEventResponse);
```
because `submitEventResponse` has `this` bound correctly because it is using `func = () => {` instead of `func() {`

Reply:
in theory that seems to work. thx